napari/resources/stylesheet.qss

```diff
@@ -1000,22 +1000,147 @@ QScrollBar::sub-page {
   qproperty-barColor: {{ highlight }};
   qproperty-handleColor: {{ highlight }};
   qproperty-handleBorderColor: {{ highlight }};
 }
 
-QtDims > QScrollBar::handle[last_used=false]:horizontal {
+QtDimSliderWidget > QScrollBar::handle[last_used=false]:horizontal {
     background: {{ highlight }};
 }
 
-QtDims > QScrollBar::handle[last_used=true]:horizontal {
+QtDimSliderWidget > QScrollBar::handle[last_used=true]:horizontal {
     background: {{ secondary }};
 }
+
+QtDimSliderWidget > QScrollBar:left-arrow:horizontal {
+    image: url(":/icons/{{ folder }}/step_left.svg");
+}
+
+QtDimSliderWidget > QScrollBar::right-arrow:horizontal {
+    image: url(":/icons/{{ folder }}/step_right.svg");
+}
+
+
+QtPlayButton {
+    /* border: none; */
+    border-radius: 3px;
+    min-height: 10px;
+    max-height: 10px;
+    min-width: 10px;
+    max-width: 10px;
+    margin: 0px 2px;
+    padding: 2px;
+}
+
+QtPlayButton[reverse=True] {
+    image: url(":/icons/{{ folder }}/left_arrow.svg");
+}
+
+QtPlayButton[reverse=False] {
+    image: url(":/icons/{{ folder }}/right_arrow.svg");
+}
+
+QtPlayButton[playing=True] {
+    image: url(":/icons/{{ folder }}/new_image.svg");
+    background-color: rgb(153, 18, 31);
+}
+
+#QtModalPopup {
+  background: transparent;
+}
+
+#QtPopupFrame {
+  background: {{ background }};
+  border: 1px solid {{ secondary }};
+  border-radius: 5px;}
+
+#QtPopupFrame > QLabel {
+  background: {{ background }};
+  color: #aaaaaa;
+  font-size: 12px;
+}
+
+
+#QtPopupFrame > QDoubleSpinBox {
+  border: none;
+  padding: 0px;
+  min-width: 85px;
+  font-size: 11px;
+}
+
+#QtPopupFrame > QDoubleSpinBox::up-button,
+#QtPopupFrame > QDoubleSpinBox::down-button {
+    background-color: {{ background }};
+    subcontrol-origin: margin;
+    width: 20px;
+    height: 20px;
+}
+
+#QtPopupFrame > QDoubleSpinBox::up-button:pressed,
+#QtPopupFrame > QDoubleSpinBox::down-button:pressed {
+      background-color: {{ highlight }};
+}
+
+#QtPopupFrame > QDoubleSpinBox::up-button {
+    subcontrol-position: center right;
+    right: 0px;
+}
+
+
+#QtPopupFrame > QDoubleSpinBox::down-button {
+    subcontrol-position: center left;
+    left: 0px;
+}
+
+#QtPopupFrame > QDoubleSpinBox::up-arrow,
+#QtPopupFrame > QDoubleSpinBox::down-arrow {
+   width: 10px;
+   height: 10px;
+}
+
+#QtPopupFrame > QDoubleSpinBox::up-arrow {
+   image: url(":/icons/{{ folder }}/plus.svg");
+}
+
+#QtPopupFrame > QDoubleSpinBox::down-arrow {
+   image: url(":/icons/{{ folder }}/minus.svg");
+}
+#QtPopupFrame > QCheckBox {
+  background-color: {{ foreground }};
+  border-radius: 3px;
+  min-height: 16px;
+  max-height: 16px;
+}
+
+
+#QtPopupFrame > QCheckBox:checked {
+   background: {{ secondary }};
+}
+
+#QtPopupFrame > QCheckBox::indicator {
+   image: url(":/icons/{{ folder }}/right_arrow.svg");
+   width: 10px;
+   height: 14px;
+   padding-right: 2px;
+   padding-left: 3px;
+}
+#QtPopupFrame > QCheckBox::indicator:checked {
```

Comment:
This also causes the background color to change as it is inheriting from other stylesheets i think - see screenshot, 
![image](https://user-images.githubusercontent.com/6531703/69825559-8de3cc00-11c4-11ea-9bf5-8509bbbff03b.png)
you can overwrite background color here

Reply:
not sure i follow.  which lines and item background are you specifically referring to?  which part did you want to look different?

Reply:
if you compare the screenshot above with the play direction button reversed it has a light gray, whereas the play direction button forward has dark gray:
![image](https://user-images.githubusercontent.com/6531703/69827839-013e0b80-11ce-11ea-835c-7ea8d27b97c3.png)
I think this is the relevant part of the sytlesheet, but maybe wrong, and i think it's because the reverse direction is also inheriting some stylesheet from elsewhere

Reply:
do you also want to style the background of the QDoubleSpinBox to have that intermediate gray too?

Reply:
oh, I actually added that change in background on the direction checkbox to make it pop a bit more when in reverse... but I don't think it's all that intuitive!  good enough just to have it switch directions?  I don't mind the look of the fps, but probably shouldn't "go rogue" from the rest of the theme :) will change

Reply:
I personally would add an extra line to the dialog, "play direction", with that icon. It would have taken me a very long time to figure out that that's not another play/pause button.

Reply:
I agree with @jni - a dedicated line is probably most clear

Reply:
ok, got a new line for play direction, made it look more like an arrow and less like a play button, and made the styles for the spinbox more like the other buttons.  is everyone ok with the label "`frames per sec:`" ?

<img width="275" alt="Screen Shot 2019-11-29 at 10 09 34 PM" src="https://user-images.githubusercontent.com/1609449/69894935-05187d80-12f5-11ea-8903-0945656a50fc.png">

Reply:
looks great to me, both the arrow and the "frames per sec:" I guess the question is would we prefer "frames per second:" to be more explicit? I don't have strong feelings, either is fine

Reply:
maybe i lean towards "frames per second:" - might as well spell it out, it won't make the box that much bigger, and it might save someone some confusion

Reply:
@tlambert03 did you have thoughts on this change? as i said not huge deal either way

Reply:
i agree, just updated